src/test/compile-fail/issue-23024.rs

```diff
@@ -18,6 +18,7 @@ fn main()
     vfnfer.push(box h);
     println!("{:?}",(vfnfer[0] as Fn)(3));
     //~^ ERROR the precise format of `Fn`-family traits'
-    //~| ERROR wrong number of type arguments: expected 1, found 0
+    //~| ERROR E0243
+    //~| NOTE expected 1 type arguments, found 0
```

Comment:
according to irc, it's because once one NOTE is there, they must all be accounted for, should I remove mine or add the 4 missing?